src/github/api/client.rs

```diff
@@ -266,19 +275,12 @@ impl GithubRepositoryClient {
             .html_url
             .as_ref()
             .map(|url| url.to_string())
-            .unwrap_or_else(|| {
-                format!(
-                    "{}/{}/{}",
-                    base_github_html_url(),
-                    self.name().owner,
-                    self.name().name
-                )
-            });
+            .unwrap_or_else(|| format!("{}/{}", base_github_html_url(), self.repository(),));
         format!("{html_url}/actions/runs/{run_id}")
     }
 
     fn format_pr(&self, pr: PullRequestNumber) -> String {
-        format!("{}/{}/{}", self.name().owner(), self.name().name(), pr)
+        format!("{}/{}", self.repository(), pr)
```

Comment:
Let's hope no one changes the `Display` of `GithubRepoName` in the future :upside_down_face: But it should be fine to depend on it.

Reply:
Let's hope we're be notified by a test failure when this happen 🤣